src/workHome/TextServlet.java

```diff
@@ -0,0 +1,39 @@
+package workHome;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/textCounter")
+public class TextServlet extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        request.setCharacterEncoding("UTF-8");
+
+        String text = request.getParameter("text2count");
+
+        System.out.println(text);
+
+        CharacterCounter chCounter = new CharacterCounter(text);
+
+        response.setCharacterEncoding("UTF-8");
+        response.setContentType("text/html");
+        PrintWriter writer = response.getWriter();
+
+        writer.println("<h1>Przelicznik tekstu:</h1>");
+        writer.println(text + "<br/>");
+        writer.println("<h1>Ilość słów: " + Utils.wordCounter(text) + "</h1><br/>");
```

Comment:
Zrobiłeś 2 klasy utilsowe, ale tak patrząc na to to wydaje mi się że łatwiej i ładniej by było jakby to było w jednej klasie a nie dwóch osobnych. No i nazwa zamiast `Utils` lepiej dać bardziej znaczącą nazwę, która wskazuje na przeznaczenie danej klasy pomocniczej np `StringUtils`

Reply:
Tyle że w tym moim pomyśle musiałbyś za każdym razem przeliczać dany atrytub np. spację, czy charactery